spec/index.html

```diff
@@ -369,11 +369,13 @@ <h3>Terms defined by this specification</h3>
         of nodes and quads such that the first quad includes
         the starting node as an component, and the last quad includes
         the ending node as an component with each quad in the path
-        containing both the preceding and following nodes.</dd>
+        containing both the preceding and following nodes. Gossip paths are
+        encoded and operated on in the <a>RDFC-1.0</a> algorithm as
+        strings.</dd>
```

Comment:
Nothing here *defines* `gossip path`, and it is not clear whether such a thing is only found between two `blank [i.e., unnamed] nodes`, or also exists between `named nodes`.

Reply:
A [gossip path is defined in the Terminology Section](https://www.w3.org/TR/rdf-canon/#dfn-gossip-path).

Reply:
*Now*, there is a definition there. The previous "definition" was circular, as it used `gossip path` in its "definition".